src/docs/metrics/social-volume/index.md

```diff
@@ -1,197 +1,166 @@
 ---
-title: Social volume metrics
-author: Ivan Klimuk
-date: 2019-09-19
+title: Social Volume
+author: Ivan
```

Comment:
will be nice to have short description for all metrics. It very useful for seo and demonstrably for user quick view on link

Reply:
😨